packages/framework/src/Component/Elasticsearch/IndexFacade.php

```diff
@@ -126,6 +126,33 @@ public function export(AbstractIndex $index, IndexDefinition $indexDefinition, O
         $this->sqlLoggerFacade->reenableLogging();
     }
 
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Elasticsearch\AbstractIndex $index
+     * @param \Shopsys\FrameworkBundle\Component\Elasticsearch\IndexDefinition $indexDefinition
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     */
+    public function exportChanged(AbstractIndex $index, IndexDefinition $indexDefinition, OutputInterface $output): void
+    {
+        $output->writeln(sprintf(
+            'Exporting changed data of "%s" on domain "%s"',
+            $indexDefinition->getIndexName(),
+            $indexDefinition->getDomainId()
+        ));
+
+        $progressBar = $this->progressBarFactory->create($output, $index->getChangedCount($indexDefinition->getDomainId()));
+
+        $lastProcessedId = 0;
+        while (($changedIdsBatch = $index->getChangedIdsForBatch($indexDefinition->getDomainId(), $lastProcessedId, $index->getExportBatchSize())) !== []) {
```

Comment:
I see... I added chunks in c1fc51db2, but in most cases, it probably will not be used.
I intentionally did not change the exportChangedMethod because I want to select IDs in batch from the DB and not all change. 

I see the chunk in `exportIds` more like a safety net.